keras_nlp/models/albert/albert_presets.py

```diff
@@ -0,0 +1,114 @@
+            "vocabulary_size": 30000,
+            "num_layers": 12,
+            "num_heads": 12,
+            "num_groups": 1,
+            "num_inner_repetitions": 1,
+            "embedding_dim": 128,
+            "hidden_dim": 768,
+            "intermediate_dim": 3072,
+            "dropout": 0.0,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
+        "preprocessor_config": {},
+        "description": (
+            "Base size of ALBERT where all input is lowercased. "
+            "Trained on English Wikipedia + BooksCorpus."
+        ),
+        "weights_url": "https://drive.google.com/uc?export=download&id=1RzTTa8nMcBc84nARvJmHal5SndpKbDUa",
+        "weights_hash": "b83ccf3418dd84adc569324183176813",
+        "spm_proto_url": "https://drive.google.com/uc?export=download&id=1-0C5mWLPRxNaY3yuzdqut-Wi7VaKo2gX",
+        "spm_proto_hash": "73e62ff8e90f951f24c8b907913039a5",
+    },
+    "albert_large_en_uncased": {
+        "config": {
+            "vocabulary_size": 30000,
+            "num_layers": 24,
+            "num_heads": 16,
+            "num_groups": 1,
+            "num_inner_repetitions": 1,
+            "embedding_dim": 128,
+            "hidden_dim": 1024,
+            "intermediate_dim": 4096,
+            "dropout": 0,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
+        "preprocessor_config": {},
+        "description": (
+            "Large size of ALBERT where all input is lowercased. "
+            "Trained on English Wikipedia + BooksCorpus."
+        ),
+        "weights_url": "https://drive.google.com/uc?export=download&id=1PfMgFPNIb4K9hiLx1Ik5Qcgol_3pOsLE",
+        "weights_hash": "c7754804efb245f06dd6e7ced32e082c",
+        "spm_proto_url": "https://drive.google.com/uc?export=download&id=1-0C5mWLPRxNaY3yuzdqut-Wi7VaKo2gX",
+        "spm_proto_hash": "73e62ff8e90f951f24c8b907913039a5",
+    },
+    "albert_extra_large_en_uncased": {
+        "config": {
+            "vocabulary_size": 30000,
+            "num_layers": 24,
+            "num_heads": 16,
+            "num_groups": 1,
+            "num_inner_repetitions": 1,
+            "embedding_dim": 128,
+            "hidden_dim": 2048,
+            "intermediate_dim": 8192,
+            "dropout": 0,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
+        "preprocessor_config": {},
+        "description": (
+            "Extra Large size of ALBERT where all input is lowercased. "
+            "Trained on English Wikipedia + BooksCorpus."
+        ),
+        "weights_url": "https://drive.google.com/uc?export=download&id=1cFQuP3QdHZ7JIA62fQxG8f9kKnv4CR-D",
+        "weights_hash": "beff185687874df846fda98386fcab29",
+        "spm_proto_url": "https://drive.google.com/uc?export=download&id=1-0C5mWLPRxNaY3yuzdqut-Wi7VaKo2gX",
+        "spm_proto_hash": "73e62ff8e90f951f24c8b907913039a5",
+    },
+    "albert_extra_extra_large_en_uncased": {
```

Comment:
Extra large indeed 😮

Reply:
lolol, i guess any naming decision you make will come and bite you in some cases.

Reply:
😂